### ISSUE TITLE
Variable star - classified as SN 

### ISSUE THREAD

**Issue by smarttgit** [thespacedoctor/sherlock#51]
https://star.pst.qub.ac.uk/sne/atlas4/candidate/1175638670203248700/

This has a NED entry which is why it gets a SN classification. 

But it also has a USNO, GAIA, PPMXL and GSC entry - so why is it getting through as a SN ?

**Comment by thespacedoctor** [comment]
The Gaia DR1 star is 4.36'' separated from the reported ATLAS transient location, too far for me to believe they are associated, esp. in this crowded field.

However the PPMXL catalogue has a source 0.316'' separated.

**Comment by thespacedoctor** [comment]
I have now PPMXL installed and ran Sherlock again on this source (ignore duplicate rows -- I'll fix these):

```bash
Suggested Associations:
+-------------------+-------+------------+----------------------------------------------------+-----------------------------+------------------------+---------------------------+-------------+--------------+-------------------+--------------------------+------------------+-----------+-------+--------+------------+---------+-----------------------------+
| association type  | rank  | rankScore  | catalogue table name                               | catalogue object id         | catalogue object type  | catalogue object subtype  | raDeg       | decDeg       | separationArcsec  | physical separation kpc  | direct distance  | distance  | z     | Mag    | MagFilter  | MagErr  | classification reliability  |
+-------------------+-------+------------+----------------------------------------------------+-----------------------------+------------------------+---------------------------+-------------+--------------+-------------------+--------------------------+------------------+-----------+-------+--------+------------+---------+-----------------------------+
| VS                | 1     | 1000.72    | The PPMXL Catalog Of Positions And Proper Motions  | 4860862025170281671         | star                   |                           | 269.161208  | -20.546926   | 0.32              |                          |                  |           |       | 18.66  | R          |         | synonym                     |
| VS                | 2     | 1000.72    | The PPMXL Catalog Of Positions And Proper Motions  | 4860862025170281671         | star                   |                           | 269.161208  | -20.546926   | 0.32              |                          |                  |           |       | 18.66  | R          |         | synonym                     |
| VS                | 3     | 2000.72    | The PPMXL Catalog Of Positions And Proper Motions  | 4860862025170281671         | star                   |                           | 269.161208  | -20.546926   | 0.32              |                          |                  |           |       | 18.66  | R          |         | association                 |
| VS                | 4     | 2000.72    | The PPMXL Catalog Of Positions And Proper Motions  | 4860862025170281671         | star                   |                           | 269.161208  | -20.546926   | 0.32              |                          |                  |           |       | 18.66  | R          |         | association                 |
| SN                | 5     | 3000.33    | NED                                                | SSTSL2 J175638.68-203248.8  | galaxy                 | IrS                       | 269.161208  | -20.54689    | 0.23              |                          |                  |           |       |        |            |         | annotation                  |
| UNCLEAR           | 6     | 3002.01    | GSC v2.3                                           | S803081029                  | unknown                | 3                         | 269.161135  | -20.5472001  | 1.31              |                          |                  |           |       | 19.84  | B          | 0.37    | annotation                  |
| VS                | 7     | 3004.86    | Gaia DR1                                           | 4071084158615336192         | star                   |                           | 269.161597  | -20.548033   | 4.36              |                          |                  |           |       | 18.40  | G          |         | annotation                  |
+-------------------+-------+------------+----------------------------------------------------+-----------------------------+------------------------+---------------------------+-------------+--------------+-------------------+--------------------------+------------------+-----------+-------+--------+------------+---------+-----------------------------+
```

So PPMXL fixes the issue for this source, but I'm still a little concerned about galaxy contamination in PPMXL.

**Comment by smarttgit** [comment]
ok, I'll also look at the documentation.

**Comment by thespacedoctor** [comment]
There is colour information we could filter/cut on.

**Comment by smarttgit** [comment]
This is V1267 Her - 13 mag variable star 
https://star.pst.qub.ac.uk/sne/atlas4/candidate/1172906761274923600/

But SDSS classifies it incorrectly is a galaxy.  Sherlock should be picking this up as a variable star initially and chucking it out. Now sure why it has come through this far in the processing, should have been binned before getting to SDSS classification

**Comment by smarttgit** [comment]
Same issue 
https://star.pst.qub.ac.uk/sne/atlas4/candidate/1140535790145147000/

**Comment by thespacedoctor** [comment]
Fixes in issue #96 fix both [1172906761274923600](https://star.pst.qub.ac.uk/sne/atlas4/candidate/1172906761274923600/) (now VS) and [1140535790145147000](https://star.pst.qub.ac.uk/sne/atlas4/candidate/1140535790145147000/) (VS).
